### ISSUE TITLE
How to use?

### ISSUE THREAD

**Issue by hmawla** [hofmannsven/laravel-brevo#4]
How to use this package? There are only installation instructions!

**Comment by ManicardiFrancesco** [comment]
I've rapidly looked at the code and this doesn't seem to be usable as a mail driver in laravel, it's just a bunch of config files and basically is saving you the boilerplate code for .env and config and service provider

**Comment by spyesx** [comment]
Agree, this project would deserve a little bit more documentation.
A wrapper is cool but how to use it would be good.

**Comment by hmawla** [comment]
Exactly yes, I never figured out how to use it, so I returned to the original library.

**Comment by spyesx** [comment]
I could use it but as @ManicardiFrancesco said, this is just a boilerplate for configuration.

Example:

```php
$brevo = new \Brevo\Client\Api\AccountApi(
    new \GuzzleHttp\Client(),
    app('brevo')->getConfiguration()
);

$result = $brevo->getAccount();
var_dump($result);
```

**Comment by lucasbaire** [comment]
> I could use it but as @ManicardiFrancesco said, this is just a boilerplate for configuration.
> 
> Example:
> 
> ```
> $brevo = new \Brevo\Client\Api\AccountApi(
>     new \GuzzleHttp\Client(),
>     app('brevo')->getConfiguration()
> );
> 
> $result = $brevo->getAccount();
> var_dump($result);
> ```

 Thanks for your commment, it helped me !
 
 Note that you can use the Facade that the package contains instead of directly accessing the container, to make the most use of what is in the package.

Adding the service provider in config/app.php :
```
    'providers' => ServiceProvider::defaultProviders()->merge([
        /*
         * Package Service Providers...
         */
         Hofmannsven\Brevo\BrevoServiceProvider::class,
         
```

Then it is just a bit simpler :
```

 $brevo = new \Brevo\Client\Api\AccountApi(
     new \GuzzleHttp\Client(),
     Brevo::getConfiguration()
);
```
